perception/autoware_crosswalk_traffic_light_estimator/README.md

```diff
@@ -8,16 +8,16 @@
 
 ### Input
 
-| Name                                 | Type                                             | Description        |
-| ------------------------------------ | ------------------------------------------------ | ------------------ |
-| `~/input/vector_map`                 | `autoware_map_msgs::msg::LaneletMapBin`          | vector map         |
-| `~/input/route`                      | `autoware_planning_msgs::msg::LaneletRoute`      | route              |
+| Name                                | Type                                                    | Description        |
+| ----------------------------------- | ------------------------------------------------------- | ------------------ |
+| `~/input/vector_map`                | `autoware_map_msgs::msg::LaneletMapBin`                 | vector map         |
+| `~/input/route`                     | `autoware_planning_msgs::msg::LaneletRoute`             | route              |
```

Comment:
Could you describe this message is optional?

Reply:
It's added here: It's added: https://github.com/autowarefoundation/autoware.universe/commit/e529247b56d761aa95e1c08828fc975a03d2d57b#diff-5a2a13b8e47e19b81499e006c7fdbf8993e9d94ecbc5d78f03ffed824274533cR28